# lift_endstops
# Status of the 8 Endstops at the top of the lifts
# 0 = lift is up ; 1 = lift is not up
int8 lift_end0
int8 lift_end1
int8 lift_end2
int8 lift_end3
int8 lift_end4
int8 lift_end5
int8 lift_end6
int8 lift_end7